source/module_base/test/math_polyint_test.cpp

```diff
@@ -0,0 +1,134 @@
+#include"../math_polyint.h"
+#include"gtest/gtest.h"
+#include"../realarray.h"
+#include<math.h>
+
+#define doublethreshold 1e-9
+
+/************************************************
+*  unit test of class PolyInt
+***********************************************/
+
+/**
+ * This unit test is to verify the accuracy of 
+ * interpolation method on the function sin(x)/x
+ * with a interval of 0.01.
+ * sin(x)/x is one of the solution of spherical bessel
+ * function when l=0.
+ * 
+ * - Tested function:
+ *      - 4 types of Polynomial_Interpolation 
+ *      - Polynomial_Interpolation_xy
+ */
+
+
+class bessell0 : public testing::Test
+{
+    protected:
+    
+    int TableLength = 400;
+    double interval = 0.01;
+    ModuleBase::realArray table3,table4;
+    ModuleBase::realArray y3;
+    double *tablex = new double[TableLength];
+    double *tabley = new double[TableLength];
+
+    double Func(double x) {return sin(x)/x;}
```

Comment:
`Func` is really a bad name for function. Did you mean [`sinc`](https://en.wikipedia.org/wiki/Sinc_function) for `sin(x)/x`?

Reply:
Sorry, I didn't realize this **function** is a solution to Bessel function